### PR TITLE
refactor: updates latest prices layout

### PR DESCRIPTION
**Pull request by kelsos**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/inputs/AssetSelect.vue [hunk 1]
```diff
@@ -267,6 +267,9 @@ const listeners = useListeners();
         />
       </div>
     </template>
+    <template #prepend>
+      <slot name="prepend" />
+    </template>
```
Comment (lukicenturi) [inline frontend/app/src/components/inputs/AssetSelect.vue:272]: image

`#prepend-inner` looks better I think (?)

Comment (kelsos) [reply frontend/app/src/components/inputs/AssetSelect.vue:272]: It does but it also pushes the label in the asset select itself when there is a selection so using it is a bit more complicated

**Review by lukicenturi — APPROVED** [review]
LGTM!

File: frontend/app/src/components/inputs/AssetSelect.vue (hunk 1 above)
Comment (kelsos) [reply frontend/app/src/components/inputs/AssetSelect.vue:272]: We can do it in a new PR at some point, but I didn't want to mess with the AssetSelect css 😢

**Pull request merged**